corehq/apps/export/utils.py

```diff
@@ -608,7 +609,8 @@ def migrate_domain(domain, dryrun=False, force_convert_columns=False):
                 )
             except Exception as e:
                 print('Failed parsing {}: {}'.format(old_export['_id'], e))
-                raise e
+                exc_type, exc_value, exc_traceback = sys.exc_info()
+                raise exc_type, exc_value, exc_traceback
```

Comment:
ah i didn't know that i was getting incorrect tracebacks and this fixed it, will test out with no args